app/models/project_step.rb

```diff
@@ -81,6 +81,10 @@ def logs_count
     project_logs.count
   end
 
+  def latest_logs(limit = 3)
+    project_logs.order(date: :desc, updated_at: :desc).limit(limit)
```

Comment:
As this is being sorted by date and time, it is unlikely that updated_at will be needed.